Functions.Templates/Templates/SqlInputBinding-JavaScript/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "defaultFunctionName": "SqlInputBinding",
+  "description": "$SqlInputBinding_description",
+  "name": "SQL Input Binding",
+  "language": "JavaScript",
+  "category": [
+    "$temp_category_core",
+    "$temp_category_dataProcessing"
+  ],
+  "categoryStyle": "sql",
+  "enabledInTryMode": false,
+  "userPrompt": [
+    "commandText",
+    "commandType",
+    "parameters",
```

Comment:
I think we should remove the `parameters` and `commandType` prompts to simplify things - we don't want to confuse new people and there isn't a good way to tell users what those values mean or how to use them. 

@dzsquared Does that seem reasonable?